webknossos/webknossos/client/api_client/models.py

```diff
@@ -49,13 +49,21 @@ class ApiBoundingBox:
     depth: int
 
 
+@attr.s(auto_attribs=True)
+class ApiAdditionalAxes:
```

Comment:
```suggestion
class ApiAdditionalAxis:
```
(I think this is singular)